web/src/components/molecules/WorkspaceSettings/DangerZone.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from "@emotion/styled";
+import { useCallback } from "react";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import ContentSection from "@reearth-cms/components/atoms/InnerContents/ContentSection";
+import Modal from "@reearth-cms/components/atoms/Modal";
+import { useT } from "@reearth-cms/i18n";
+
+export type Props = {
+  onWorkspaceDelete: () => Promise<void>;
+};
+
+const DangerZone: React.FC<Props> = ({ onWorkspaceDelete }) => {
+  const t = useT();
+  const { confirm } = Modal;
+
+  const handleWorkspaceDeleteConfirmation = useCallback(() => {
+    confirm({
+      title: t("Are you sure you want to delete this workspace?"),
+      icon: <Icon icon="exclamationCircle" />,
+      onOk() {
+        onWorkspaceDelete();
+      },
+    });
+  }, [confirm, onWorkspaceDelete, t]);
+
+  return (
+    <ContentSection title={t("Danger Zone")} danger>
+      <Title>{t("Remove Workspace")}</Title>
+      <Text>
+        {t(
+          "Permanently remove your Workspace and all of its contents from the Re:Earth CMS. This action is not reversible – please continue with caution.",
```

Comment:
Permanently removes the Workspace and all of its contents from Re:Earth CMS. This action is not reversible, so please continue with caution.

Reply:
@KaWaite , updated.